experiments/ClimaEarth/components/ocean/eisenman_seaice.jl

```diff
@@ -79,8 +79,13 @@ function EisenmanIceSimulation(
         thermo_params = thermo_params,
         dss_buffer = CC.Spaces.create_dss_buffer(Y),
     )
-    problem = SciMLBase.ODEProblem(ode_function, Y, Float64.(tspan), cache)
-    integrator = SciMLBase.init(problem, ode_algo, dt = Float64(dt), saveat = Float64.(saveat), adaptive = false)
+    if typeof(dt) isa AbstractFloat
+        dt = Float64(dt)
+        tspan = Float64.(tspan)
+        saveat = Float64.(saveat)
+    end
```

Comment:
Should we instead always convert `span` and `saveat` to the type of `dt` so we always use consistent time types?
```suggestion
    tspan = typeof(dt).(tspan)
    saveat = typeof(dt).(saveat)
```

Reply:
We can also remove the default `dt` value from this function since we always pass it in

Reply:
If `dt` is an `ITime`, then `tspan = typeof(dt).(tspan)` doesn't work.